Execute e2e cypress tests as part of integration tests

sdorra opened the pull request:
## Proposed changes

This pr will add the execution of the '@scm-manager/e2e-tests' package as part of scm-it.
To see a failing build, with screenshots and videos have a look at build [#4](https://oss.cloudogu.com/jenkins/job/scm-manager-github/job/scm-manager/job/feature%252Fe2e_pipeline/4/) of the `feature/e2e_pipeline` branch.

### Your checklist for this pull request

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [X] PR is well described
- [X] Target branch is not master (in most cases develop should bet the target of choice) 
- [X] Code does not conflict with target branch
- [X] New code is covered with unit tests

### Checklist for branch merge request (not required for forks)

- [x] Branch is green/blue on [Jenkins](https://oss.cloudogu.com/jenkins/)
- [x] Quality Gate passed on [SonarQube](https://sonarcloud.io/organizations/scm-manager/projects)

eheimbuch reviewed (commented):
Do we really want to save the videos if the build is successful? Maybe we should only save the videos of failed tests?

eheimbuch reviewed (commented):
The video quality is quite terrible. Is it possible to increase the frame rate so we can see what is actually going on? If this bloath the size too much we can decrease the resolution.

sdorra commented:
> Do we really want to save the videos if the build is successful? Maybe we should only save the videos of failed tests?

It seams that this feature is reserved for the dashboard service of cypress:

* https://docs.cypress.io/guides/references/configuration.html#Videos
* https://github.com/cypress-io/cypress/issues/2522

sdorra commented:
> The video quality is quite terrible. Is it possible to increase the frame rate so we can see what is actually going on? If this bloath the size too much we can decrease the resolution.

I think the quality is good enough to see the things which matter. However there is no option to increase the frame rate nor to decrease the resolution. The only option which affects the quality is the `compression`: https://docs.cypress.io/guides/references/configuration.html#Videos

sdorra commented:
@eheimbuch i've disabled compression, removed videos which does not contain failing tests and split the rest up into videos for each test (instead of the whole suite) and keep only those which are failing.

eheimbuch commented on scm-ui/e2e-tests/src/index.js:
```diff
@@ -0,0 +1,86 @@
+  }
+};
+
+const createOuputFile = test => {
```
typo
